README.md

```diff
@@ -4,97 +4,164 @@
 
    * [snowdrop-automation-client](#snowdrop-automation-client)
       * [Table of contents](#table-of-contents)
-      * [Basics](#basics)
-      * [Running](#running)
-         * [Prerequisites](#prerequisites)
-         * [Launch application](#launch-application)
-      * [Install specific nvm version](#install-specific-nvm-version)
-      * [Miscellaneous](#miscellaneous)
-      * [Run in a container](#run-in-a-container)
-         * [buildah/podman](#buildahpodman)
-            * [Building the container](#building-the-container)
-            * [Running the container](#running-the-container)
-            * [k8s](#k8s)
-         * [Docker](#docker)
-            * [Build with docker-compose](#build-with-docker-compose)
-            * [Run with docker](#run-with-docker)
+      * [Introduction](#introduction)
+      * [Available commands](#available-commands)
+      * [Configuration](#configuration)
+      * [Running the client locally](#running-the-client-locally)
+      * [Running the client in a container](#running-the-client-in-a-container)
+      * [Available npm scripts](#available-npm-scripts)
       * [References](#references)
 
 ## Introduction
 
-Contains automation used by the snowdrop team
+This project implements automation commands used by the Snowdrop team.
+The commands are executed with Slack messages and integrate with Atomist SDM.
 
-## Basics
+This application can run in any environment that can run a Node.js application.
+Including a server in a private network.
 
-The automation client is a NodeJS application that communicates with the Atomist API in order to
-perform operations an receive events.
 
-This application can anywhere a NodeJS application can run. One thing to note is that it can also
-run inside a private network since it uses Websockets to communicate with the Atomist API.
+## Available commands
 
-## Running
+### Update an example property
 
-### Prerequisites
+This command will update a `pom.xml` property of all of the Snowdrop example applications.
 
-* Properly configure your environment in order for Atomist to be able to work. See [this](https://docs.atomist.com/developer/prerequisites/)
+Send the following Slack message to Atomist:
+```
+update boosters maven property
```

Comment:
Maybe we should rename `boosters` to `examples` at some point… :)

Reply:
Yep, this was my bad a few months ago. I renamed all the main commands but must have missed this one for some reason. I'll keep it in the readme for now and will change it one I refactor the command at some point in the future.

Reply:
Gotcha.